.vscode/launch.json

```diff
@@ -6,9 +6,12 @@
       "type": "node2",
       "request": "launch",
       "name": "run bootstrapper",
-      "program": "${workspaceRoot}/src/autorest/app.js",
-      "cwd": "${workspaceRoot}",
-      "preLaunchTask": "build/typescript"
+      "program": "${workspaceRoot}/src/autorest-core/app.js",
```

Comment:
don't commit that or create a new task since this is not "run bootstrapper" anymore